Prepare for release 1.0.7 (#373)
Azure/iotedge-lorawan-starterkit: Docs/upgrade.md

# Upgrade LoRaWAN to a new version
<!-- allow github to generate 'copy' buttons for coded URLs -->
<!-- markdownlint-disable MD040 -->
## Release 1.0.7

To update from version 1.0.6, 1.0.5, 1.0.4 or 1.0.3 you can follow the below instructions. If you want to update manually from a version prior to 1.0.3, please refer to the instructions in the [Release 1.0.3](#Release-1.0.3) section below.

### Update the IoT Edge security daemon when upgrading from IoT Edge 1.1 (release prior to 1.0.6)

Since release 1.0.6, the starter kit uses Azure IoT Edge version 1.2 which includes major changes the the IoT Edge Security daemon. Please follow this documentation to [Update IoT Edge](https://docs.microsoft.com/azure/iot-edge/how-to-update-iot-edge?view=iotedge-2020-11&tabs=linux) to upgrade Azure IoT Edge to 1.2.

> ### Updating from release post 1.0.3

Reviewer: ```suggestion
### Updating IoT Edge LoRaWAN starterkit from release post 1.0.3
```

# Upgrade LoRaWAN to a new version
<!-- allow github to generate 'copy' buttons for coded URLs -->
<!-- markdownlint-disable MD040 -->
## Release 1.0.7

> > Note that this version switch IoT Edge to use version 1.2 of IoT Edge that include major changes on the edge. Please follow this [documentation](https://docs.microsoft.com/azure/iot-edge/how-to-update-iot-edge?view=iotedge-2020-11&tabs=linux) to upgrade IoT Edge to 1.2.

Reviewer: nit: not sure how you do this usually, but the official name would be Azure IoT Edge.

> > Note that this version switch IoT Edge to use version 1.2 of IoT Edge that include major changes on the edge. Please follow this [documentation](https://docs.microsoft.com/azure/iot-edge/how-to-update-iot-edge?view=iotedge-2020-11&tabs=linux) to upgrade IoT Edge to 1.2.

Reviewer: ```suggestion
> This version starts using Azure IoT Edge 1.2 which includes major changes. Please follow [Update IoT Edge](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-update-iot-edge?view=iotedge-2020-11&tabs=linux) to upgrade to Azure IoT Edge 1.2.
```

Author: Right, much better. I will just keep the URL langage agnostic

Reviewer: By URL language you mean the title of the document? It's considered best practice to use the title in the links.

To update from version 1.0.6, 1.0.5, 1.0.4 or 1.0.3 you can follow the below instructions. If you want to update manually from a version prior to 1.0.3, please refer to the instructions in the [Release 1.0.3](#Release-1.0.3) section below.

> ### Updating from release after 1.0.3

Reviewer: nit
```suggestion
### Updating from release post 1.0.3
```
